CMake/External_VXL.cmake

```diff
@@ -20,11 +20,13 @@ add_package_dependency(
   )
 
 # libgeotiff
-add_package_dependency(
-  PACKAGE VXL
-  PACKAGE_DEPENDENCY libgeotiff
-  PACKAGE_DEPENDENCY_ALIAS GEOTIFF
-  )
```

Comment:
I think you can enable the same thing by using the OPTIONAL option in add_package_dependency, right?

Reply:
I'm not sure on the exact particulars of the script, ideally though we want to force geotiffs use if it is turned on at least, does the script do that?